### DISCUSSION TITLE
가게 정보 및 리뷰 보기 페이지 관련 아이디어 모아보기

### DISCUSSION THREAD

**Discussion by labydin** [SwiftBreakers/TTEOPPOKKI4U#80] [General]
리뷰 보기 페이지에 작성한 유저의 프로필 사진과 닉네임(시간내 가능하다면), 리뷰 작성 날짜가 더해지면 좋을 듯 합니다. 
아래는 리뷰 보기 페이지 레퍼런스입니다. 사진처럼 가게 이름과 제목 등의 위치를 바꾸는 것이 괜찮아 보였습니다.
![IMG_3467](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/5bf98593-39c3-4228-9d33-6f8d2ec4ed77)
![IMG_3468](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/2b7c9460-9648-47d6-9d84-af38940dddc2)

가게 정보 페이지에도 리뷰 작성한 날짜가 보이는 게 좋을 듯 합니다. 제 친구는 너무 오래된 리뷰는 안 믿더라구요..

**Comment by labydin** [comment]
빵동여지도 설정 부분이 저희가 원하는 정보들과 거의 비슷하게 들어가 있습니다. 내용을 참고하면 좋을 듯 하여 올려봅니다. 특히 github에 써놓은 점이 새로웠어요.
![IMG_3469](https://github.com/SwiftBreakers/TTEOPPOKKI4U/assets/142779268/2447aa9c-6282-4505-8c4d-121aebe992b9)

**Comment by jinmoon23** [comment]
`중간발표 들으면서 우리 프로젝트에 적용하면 좋을 것 같은 아이디어를 정리했습니다. 이것도 선택적으로 고려해주시면 좋을 것 같아요!` 

- 유저플로우에 따라 리워드를 제공하는 점 → 참신
- 우리 어플도 리뷰수(유저의 기여)에 따라 떡볶퀸 / 떡볶킹 호칭을 수여하는 등 리워드 구성하면 좋을듯
- 어플의 오류 및 기능 제안을 건의할 수 있는 기능
- 앱 사용에 대한 동기부여를 위해 애니메이션에 상당한 공을 들인 듯
- 플랜 페이지 → [FloatingPanel 라이브러리](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://github.com/scenee/FloatingPanel&ved=2ahUKEwjUufvu-eGGAxVJg68BHb7QD8wQFnoECBEQAQ&usg=AOvVaw3HPkH6hJnIcsgvnrT8jTmo) 좋다! → MapVC에서 바텀시트 구성에 사용하면 좋을 것 같습니다.
- 페르소나 타겟 참신
- 앱 첫 구동시 온보딩 페이지 구성 → 이후 로그인 페이지
- 온보딩 페이지 구성은 유저 입장에서 굉장히 좋은 접근법 같습니다. 저희도 벤치마킹하죠 ㅎㅎ
- 소셜(커뮤니티)기능에서 유저 신고 및 차단기능 구현
- 차단한 유저를 유저가 확인할 수 있고 이후에 해제도 할 수 있는 별도 페이지 구성
- 인기글에 대한 별도의 알고리즘을 구성
- 여행 메이트 추가 → 우리도 떡볶이 친구추가 기능이 필요할 것 같습니다.
- 리뷰 → 제목과 내용이 필수로 들어가야 한다면 내용이 없을때와 있을때 버튼 색상을 달리 하는 것이 좋습니다.
